### PR TITLE
Added Serial Com

### PR DESCRIPTION
**Pull request by Vinuyans**
- Class to communicate between the BB and raspi
-  Implemented SEND, TAKE, and all 3 INFO functions

### CODE REVIEW

File: coms.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+from serial import Serial
+import time
+import re
+
+
+class SerialComs:
```
Comment (Marian-Hristov) [inline coms.py:6]: btw, what is our plan for the comms in the future? do we want to enable multiple async comms?

File: test.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        elif command == "istat":
+            print(com.info_station())
+        elif command == "ibb":
+            print(com.info_bb())
+        elif command == "icolor":
+            print(com.info_color())
+        elif command == "take":
+            print(com.take("PURPLE", 1))
+        else:
+            break
```
[the remark this answers is not in the document]
Comment (Vinuyans) [reply test.py:23]: This was only a test script and I l=only kept it to give an example of how to call the functions for easier integration later on so I don't think it's necessary

File: coms.py (hunk 1 above)
Comment (Vinuyans) [reply coms.py:6]: We can only ever be connected to one station at a time so we wouldn't ever need to send 2 messages at the same time so no need for async

File: coms.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        return self._buffer_com(f"INFO:S\n")
+
+    def info_bb(self) -> str:
+        return self._buffer_com(f"INFO:B\n")
+
+    def info_color(self) -> str:
+        return self._buffer_com(f"INFO:C\n")
+
+    def secret(self, secret_msg: str) -> None:
+        # TODO find out what the msg is
```
[the remark this answers is not in the document]
Comment (nMWGecko) [reply coms.py:36]: the github is not private do not add

**Review by nMWGecko — APPROVED** [review]

**Pull request merged**